mmv1/templates/terraform/examples/resource_manager_capability.tf.tmpl

```diff
@@ -0,0 +1,16 @@
+resource "google_folder" "folder" {
+  provider         = google-beta
+  display_name     = "{{index $.Vars "display_name"}}"
+  parent           = "organizations/{{index $.TestEnvVars "org_id"}}"
+}
+resource "time_sleep" "wait_60s" {
+  depends_on = [google_folder.folder]
+  create_duration = "60s"
+}
+resource "google_resource_manager_capability" "{{$.PrimaryResourceId}}" {
+  provider         = google-beta
+  value            = true
+  parent           = "${google_folder.folder.name}"
+  capability_name  = "{{index $.Vars "capability_name"}}"
```

Comment:
Do we expect to have multiple capabilities in the future? My understanding is that there is only one valid capability_name at this point.